packages/components/src/internal/components/entities/EntityInsertPanel.tsx

```diff
@@ -955,6 +956,12 @@ export class EntityInsertPanelImpl extends Component<Props, StateProps> {
                     {!isLoaded && !insertModel.isError && !!insertModel.targetEntityType?.value && (
                         <LoadingSpinner wrapperClassName="loading-data-message" />
                     )}
+                    {isLoaded && (
+                        <EntityInsertGridRequiredFieldAlert
+                            type={this.capTypeTextSingular}
+                            queryInfo={queryGridModel?.queryInfo}
+                        />
+                    )}
```

Comment:
This could be included in the block below, but I guess this saves a use of `<>`.

Reply:
Updated to be within the same block.